tur/gcc-default/postinst.sh.in

```diff
@@ -2,7 +2,11 @@
 
 set -e
 
-TERMUX_ARCH="$(dpkg --print-architecture)"
+if [ "$TERMUX_PACKAGE_FORMAT" = "debian" ]; then
```

Comment:
The variable should be `TERMUX_MAIN_PACKAGE_FORMAT`, as `TERMUX_PACKAGE_FORMAT` only exists in the building system.